dlpy/tests/test_imagetable.py

```diff
@@ -206,3 +204,12 @@ def test_as_random_patches(self):
         self.assertTrue(int(out[4]) == 200)
         self.assertTrue(int(out[5]) == 200)
         self.assertTrue(int(out[6]) == 200)
+
+    def test_show(self):
```

Comment:
an example where where clause fails. we might need to catch that error. thanks

Reply:
Yes. We can either give a warning message and ignore where clause or response an error. Which is better?

Reply:
I think this one is better: give a warning message and ignore where clause